Deployment/src/Modules/Webapp/main.tf

```diff
@@ -18,7 +18,7 @@ resource "azurerm_app_service" "webapp_service" {
   tags                = var.tags
 
   site_config {
-    windows_fx_version  =   "DOTNET|6.0"
+    windows_fx_version  =   "DOTNET|8.0"
```

Comment:
GitHub won't let me put this comment in the correct place but there are two other references to "DOTNET|6.0" in this module below.